### DISCUSSION TITLE
Expand information (region of the country) in published notice titles

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#731] [Ideas]
In the publications of the notices sent through TED, in the title of the publication, in addition to the country, "the region" of the country was shown.

This information is no longer displayed with eForms.

This information about the region was useful for us to search for the notices that really interested us.

Could it be added back?

For example:

[https://ted.europa.eu/udl?uri=TED:NOTICE:639721-2023:TEXT:EN:HTML](url)

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/ccff15ba-63fa-4ee2-985c-d12d51e4e1c3)

Thanks in advance,
Alex

**Comment by ferrakaop** [comment]
Please contact the TED helpdesk to raise this issue related to the TED website.
